### ISSUE TITLE
Syntax highlighting with JSX fragments broken - possible to update latest stable build with fix?

### ISSUE THREAD

**Issue by ollie-o** [microsoft/vscode#74572]
The problem I'm facing is identical to the one mentioned in [this issue from 2017](https://github.com/microsoft/vscode/issues/39694).

The solution provided in that issue is to use the latest Insiders Build. But not everybody wants to use the Insiders Build because it is unstable.

Since this bug has been fixed in the Insiders Build for 2 years already, would it be possible to add the fix it to the latest stable build?

Thanks!

## I have added more info below
----

- VSCode Version: `1.34.0`
- OS Version: `MacOS Mojave, 10.14.5`

Steps to Reproduce:

1. Create new file with these contents:
```
const Foo = () => <>Content</>;

Bar.baz = 'this is not highlighted correctly';
```
2. In the bottom-right corner under "Select language mode", select `JavaScript React`
3. See that any code after `/>` is not highlighted correctly (see screenshot)
<img width="500" alt="Screen Shot 2019-05-29 at 12 57 12 PM" src="https://user-images.githubusercontent.com/5886107/58579728-4c706500-8211-11e9-8414-c7143662f285.png">

Does this issue occur when all extensions are disabled?: Yes

Does this issue occur for the Insiders Build?: No, not for `1.35.0-insider`

**Comment by kieferrm** [comment]
@ollie-o pretty much all changes in insiders roll over to stable on a monthly basis. Thus, if you see the same issue, the issue was either not fixed or it's a regression. Either way, pls provide detailed information that allows us to reproduce the problem.

**Comment by ollie-o** [comment]
@mjbvz @kieferrm Thanks for the explanation and quick response! I have added more details to the issue description. Thanks 👍

**Comment by mjbvz** [comment]
Closing as it is fixed in the insiders build (which will be released as the may update, 1.35)

**Comment by ollie-o** [comment]
I got the `1.35.1` release today and the bug I had reported is fixed. Thank you 👍

Note for anyone with a similar issue: it could be caused by one of your extensions. Try running VSCode with all extensions disabled by using the command `code --disable-extensions`. When you do that, you should see that the bug is fixed.
